pkgs/build-support/build-dotnet-module/default.nix

```
@@ -68,9 +68,9 @@ assert nugetDeps == null -> throw "Defining the `nugetDeps` attribute is require
 
 let
   _nugetDeps = linkFarmFromDrvs "${name}-nuget-deps" (import nugetDeps {
-    fetchNuGet = { name, version, sha256 }: fetchurl {
-      name = "nuget-${name}-${version}.nupkg";
-      url = "https://www.nuget.org/api/v2/package/${name}/${version}";
+    fetchNuGet = { pname, version, sha256 }: fetchurl {
+      name = "nuget-${pname}-${version}.nupkg";
```

Would you mind changing this? The `linkFarmFromDrvs`'s name already expresses that it contains nuget deps.
```suggestion
      name = "${pname}-${version}.nupkg";
```

This comment applies to `osu-lazer` and `jellyfin` as well.

---

yes for sure. should be fixed now.
If you approve my changes please approve the PR.